src/OpenTelemetry.Exporter.Geneva/Metrics/OtlpProtobuf/OtlpProtobufSerializer.cs

```diff
@@ -444,14 +452,76 @@ private void WriteNumberDataPoint<T>(byte[] buffer, ref int cursor, int fieldNum
             cursor += this.prepopulatedNumberDataPointAttributesLength;
         }
 
-        // TODO: exemplars.
+#if EXPOSE_EXPERIMENTAL_FEATURES
+        if (metricPoint.TryGetExemplars(out var exemplars))
+        {
+            foreach (ref readonly var exemplar in exemplars)
+            {
+                if (typeof(T) == typeof(long))
+                {
+                    this.SerializeExemplar(buffer, ref cursor, in exemplar, exemplar.LongValue, FieldNumberConstants.NumberDataPoint_exemplars);
+                }
+                else if (typeof(T) == typeof(double))
+                {
+                    this.SerializeExemplar(buffer, ref cursor, in exemplar, exemplar.DoubleValue, FieldNumberConstants.NumberDataPoint_exemplars);
+                }
+            }
+        }
+#endif
 
         var metricPointStartPosition = this.metricPointTagAndLengthIndex;
 
         // Write numberdatapoint {Repeated field}
         ProtobufSerializerHelper.WriteTagAndLengthPrefix(buffer, ref metricPointStartPosition, cursor - this.metricPointValueIndex, fieldNumber, WireType.LEN);
     }
 
+#if EXPOSE_EXPERIMENTAL_FEATURES
+    private void SerializeExemplar<T>(byte[] buffer, ref int cursor, in Exemplar exemplar, T value, int fieldNumber)
```

Comment:
Mark this method and `SerializeExemplarTags` static.

Reply:
I am planning to do a follow up and mark the methods appropriately (There are few other methods that needs change). Did not want to do it here as that requires moving things around.